Source/WebCore/rendering/mac/RenderThemeMac.mm

```diff
@@ -695,11 +695,11 @@ static Color activeButtonTextColor()
     return !style.writingMode().isHorizontal();
 }
 
-bool RenderThemeMac::isControlStyled(const RenderStyle& style, const RenderStyle& userAgentStyle) const
+bool RenderThemeMac::isControlStyled(const RenderStyle& style) const
 {
     auto appearance = style.usedAppearance();
     if (appearance == StyleAppearance::TextField || appearance == StyleAppearance::TextArea || appearance == StyleAppearance::SearchField || appearance == StyleAppearance::Listbox)
-        return style.border() != userAgentStyle.border();
+        return style.nativeAppearanceDisabled();
```

Comment:
This is technically a behavior change for apps linked before `SDKAlignedBehavior::DevolvableWidgets`.

Reply:
After more consideration, this is probably okay. The implication would be that apps linked before `SDKAlignedBehavior::DevolvableWidgets` would no longer be able to customize the background color of their text fields, text areas, and <select multiple> without also losing native appearance.

However, the control would simply devolve and their specified background color would be honored so I would not expect there to be any contrast/legibility issues. Which leaves a minor cosmetic change without impact to functionality.